StroopTest/Views/ListsPages/FormWordColorConfig.cs

```diff
@@ -278,6 +278,19 @@ private static void MoveListViewItems(ListView sender, MoveDirection direction,
 
                 sender.Items.Remove(item);
                 sender.Items.Insert(index, item);
+                sender.Clear();
+                foreach (string it in list)
```

Comment:
Eu fiz esse for pra adaptar o que não tava funcionando na hora de subir e descer itens da lista, outra sugestão de solução é bem vinda